AnkiDroid/src/main/java/com/ichi2/anki/multimedia/audio/ui/compose/AudioRecorderScreen.kt

```
@@ -0,0 +1,279 @@
+            contentDescription = stringResource(R.string.save),
+            tint = if (enabled) Color.White else Color.Gray
+        )
+    }
+}
+
+private fun formatDuration(millis: Long): String {
+    val totalSeconds = millis / 1000
+    val minutes = totalSeconds / 60
+    val seconds = totalSeconds % 60
+    val deciseconds = (millis % 1000) / 100
+    return String.format("%02d:%02d.%d", minutes, seconds, deciseconds)
```

detekt (1.23.8)

[warning] String.format("%02d:%02d.%d", minutes, seconds, deciseconds) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)